src/libraries/System.Runtime.Extensions/tests/System/Environment.GetEnvironmentVariable.cs

```diff
@@ -220,12 +220,6 @@ public void EnumerateEnvironmentVariables(EnvironmentVariableTarget target)
             bool lookForSetValue = (target == EnvironmentVariableTarget.Process)
                 || (PlatformDetection.IsWindows && PlatformDetection.IsPrivilegedProcess);
 
-            // [ActiveIssue("https://github.com/dotnet/runtime/issues/30566")]
-            if (PlatformDetection.IsWindowsNanoServer && target == EnvironmentVariableTarget.User)
```

Comment:
I guess this has to stay in SetEnvironmentVariable?

https://github.com/dotnet/runtime/blob/555369171d198c49d45051e61dee2797ab0561fb/src/libraries/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs#L19

Reply:
actually I think that should be removed 

https://github.com/dotnet/runtime/blob/555369171d198c49d45051e61dee2797ab0561fb/src/libraries/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs#L19-L23